tenants/ai-example/multi-model-serving/base/inference-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    openshift.io/display-name: fraud-detection-model
+    serving.kserve.io/deploymentMode: ModelMesh
+  name: fraud-detection-model
+  labels:
+    opendatahub.io/dashboard: 'true'
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: onnx
+        version: '1'
+      name: ''
+      resources: {}
+      runtime: multi-model-server
+      storage:
+        key: aws-connection-multi-model
+        path: models/fraud-detection-model/tenants/ai-example/multi-model-serving/models/frauddetectionmodel.onnx
```

Comment:
How difficult would it be to update the path to be something like this with the upload job?

models/fraud-detection-model/frauddetectionmodel.onnx

Reply:
updated the pathing, had to change the other examples